libevmasm/Assembly.h

```diff
@@ -174,6 +179,10 @@ class Assembly
 	std::map<util::h256, std::string> m_libraries; ///< Identifiers of libraries to be linked.
 	std::map<util::h256, std::string> m_immutables; ///< Identifiers of immutables.
 
+	/// map from sub assembly id to vector representing path to particular sub assembly
+	/// this map is used only for sub-assemblies which are not direct sub-assemblies (where path is having more than one value)
```

Comment:
After https://github.com/ethereum/solidity/pull/9235#pullrequestreview-448301922 the vector is the key:

```suggestion
	/// Map from a vector representing a path to a particular sub assembly to sub assembly id.
	/// This map is used only for sub-assemblies which are not direct sub-assemblies (where path is having more than one value).
```